cdap-examples/StreamConversion/src/test/java/co/cask/cdap/examples/streamconversion/StreamConversionTest.java

```diff
@@ -99,4 +105,20 @@ public void testStreamConversion() throws Exception {
     Assert.assertEquals(minute, results.getInt(5));
     Assert.assertFalse(results.next());
   }
+
+  private <T> T testWithRetry(Callable<T> callable, int timesToRetry,
```

Comment:
I think that won't work with current implementation of Tasks.waitFor, because Tasks.waitFor doesn't do retries upon exceptions.